README.md

```diff
@@ -1,46 +1,35 @@
-# Getting Started with Create React App
+# 🚴‍♀️ Plova 🚴‍♂️
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+[> Plova web site](http://nodeca.github.io/pica/demo/ "title text!")
 
-## Available Scripts
+```
+Demo version of famous delivery application 'Glovo'
+```
 
-In the project directory, you can run:
+App presentation using development best practices like:
 
-### `npm start`
+1. React & TypeScrypt
+2. Redux, redux-tool-kit
+3. Git and Github
+4. Deployed by netlify
+5. And more, and more.. to be continue..
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+---
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+Libraries are used in app:
 
-### `npm test`
+- React router
+- Redux toolkit
+- Auth0
+- Axios
+- Stripe
+- Normalize.css
+- Styled-components
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+---
 
-### `npm run build`
+_Special thanks for the development of the back-end for the application and mentoring in the project to my mentor, senior back-end developer Danko Andrey_
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## How to start app?
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+- Maybe this section is not needed.
```

Comment:
This section is needed and most important.

Reply:
Ok. '- Maybe this section is not needed.' - was about coming after all previous info.